src/luminol/algorithms/anomaly_detector_algorithms/exp_avg_detector.py

```diff
@@ -34,6 +34,8 @@ def __init__(self, time_series, baseline_time_series=None, smoothing_factor=None
         """
         super(ExpAvgDetector, self).__init__(self.__class__.__name__, time_series, baseline_time_series)
         self.use_lag_window = use_lag_window
+        if smoothing_factor is None:
+            smoothing_factor = 0
```

Comment:
Wouldn't it be better to change the method definition to set default smoothing_factor to 0 instead?

Reply:
Will do. This is would definitely be a cleaner way to do it.